pallets/collator-assignment/src/mock.rs

```diff
@@ -14,6 +14,7 @@
 // You should have received a copy of the GNU General Public License
 // along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
 
+use crate::CoreAllocationConfiguration;
```

Comment:
fmt (put it inside the rest of dependencies)

Reply:
*fixed*